Library/Formula/binwalk.rb

```diff
@@ -1,41 +1,21 @@
 class Binwalk < Formula
   desc "Searches a binary image for embedded files and executable code"
   homepage "http://binwalk.org/"
-  revision 1
 
   stable do
-    url "https://github.com/devttys0/binwalk/archive/v2.0.1.tar.gz"
-    sha256 "90ee8426d71e91b62dfe4a1446c457bc7835b475b28717859e275a0494403959"
-
-    # Fixes OS-X-specific issues no longer relevant in HEAD:
-    #
-    # * Fixes OS X bug in 'setup.py':
-    #   * See <https://github.com/devttys0/binwalk/commit/8278229cae4c2c354ffc5bfc3bcef0fd1d9bf2b3>
-    #     and <https://github.com/devttys0/binwalk/commit/76ff729a19552e6e58505ca64b1d4ce2325e7ac0>.
-    # * Fixes library lookup for non-standard Homebrew installations:
-    #   * See upstream issue <https://github.com/devttys0/binwalk/issues/130>
-    #     for the details.
-    #   * The fix is Homebrew-specific as it uses HOMEBREW_PREFIX, that is
-    #     implicitly replace with the actual Homebrew prefix, in the patch.
-    patch :DATA
+    url "https://github.com/devttys0/binwalk/archive/v2.1.1.tar.gz"
+    sha256 "1b70a5b03489d29f60fef18008a2164974234874faab48a4f47ec53d461d284a"
   end
 
   head do
     url "https://github.com/devttys0/binwalk.git"
-
-    option "with-capstone", "Enable disasm options via capstone"
-
-    resource "capstone" do
-      url "https://pypi.python.org/packages/source/c/capstone/capstone-3.0.2.tar.gz"
-      sha256 "b32022fe956e940f8e67c17841dd3f6f1c50a60e451f9b5ce1f4dd2e5c5b3339"
-    end
   end
 
   option "with-matplotlib", "Check for presence of matplotlib, which is required for entropy graphing support"
+  option "with-capstone", "Enable disasm options via capstone"
 
   depends_on "swig" => :build
   depends_on :fortran
-  depends_on "libmagic" => "with-python"
```

Comment:
I have to check :) I thought this to be a dependency of the former C compilation, but I might be wrong. Thx for your sharp eyes